restic/genericCommand.go

```diff
@@ -127,6 +128,9 @@ func (g *genericCommand) collectOutput(output io.ReadCloser, print bool) ([]stri
 			fmt.Println(m)
 		}
 		collectedOutput = append(collectedOutput, m)
+		if out != nil {
```

Comment:
So, we are collecting all the output in memory but we also send line after line to a channel and maybe we also write it to STDOUT (no log level/verbosity configurable?).

Not to myself: I need to suggest a better way to do this by doing some refactoring.

Reply:
See my previous comment -> new ticket.